src/libsync/clientsideencryption.cpp

```diff
@@ -1538,33 +1629,41 @@ void FolderMetadata::setupExistingMetadata(const QByteArray& metadata)
   QJsonDocument debugHelper;
   debugHelper.setObject(metadataKeys);
   qCDebug(lcCse) << "Keys: " << debugHelper.toJson(QJsonDocument::Compact);
+  
+  for (auto it = folderUsers.constBegin(); it != folderUsers.constEnd(); ++it) {
```

Comment:
why are you using an old `for` loops instead of `for (auto it : folderUsers) {` ?